kiwi/builder/disk.py

```diff
@@ -347,10 +349,12 @@ def create_disk(self) -> Result:
         luks_root = None
         if self.luks is not None:
             luks_root = LuksDevice(device_map['root'])
-            self.luks_boot_keyname = '/.root.keyfile'
+            self.luks_boot_keyname = '/root/.root.keyfile'
```

Comment:
This change here is the reason for the failed unit test. Can you explain why you are changing the location of the keyfile ? Thanks